lib/rubocop/cop/style/string_hash_keys.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of strings as keys in hashes. The use of
+      # symbols is preferred instead.
+      #
+      # @example
+      #   # bad
+      #   { 'one' => 1, 'two' => 2, 'three' => 3 }
+      #
+      #   # good
+      #   { one: 1, two: 2, three: 3 }
+      class StringHashKeys < Cop
+        MSG = 'Prefer symbols instead of strings as hash keys.'.freeze
+
+        def_node_matcher :string_hash_key?, <<-PATTERN
+          (pair (str _) _)
+        PATTERN
+
+        def on_pair(node)
+          return unless string_hash_key?(node)
+          add_offense(node.key)
+        end
+
+        private
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.source_range, ":#{node.str_content}")
```

Comment:
autocorrect may break the code if number keys are used as strings :thinking: 
`autocorrect_source("{ '1' => 'one' }")` gives `{:1 => 'one'}`

Reply:
Another case is strings containing spaces.

Reply:
I fixed it by using `#to_sym.inspect`. Let me know!

Reply:
Didn't even think about that. I think it's a good solution until we add a `SymbolNode` node extension. (Might even want to use that approach there.) :thumbsup: